gremlin-go/driver/traversal.go

```diff
@@ -67,7 +67,7 @@ func (t *Traversal) ToSet() (map[*Result]bool, error) {
 }
 
 // Iterate all the Traverser instances in the traversal and returns the empty traversal.
-func (t *Traversal) Iterate() (*Traversal, <-chan bool, error) {
+func (t *Traversal) Iterate() (*Traversal, <-chan error, error) {
 	// TODO: AN-979 This wont be needed once DriverRemoteConnection is replaced by TraversalStrategy
```

Comment:
```suggestion
	// TODO: AN-1065 This wont be needed once DriverRemoteConnection is replaced by TraversalStrategy
```
Please also change the ticket number in the comment to AN-1065 for line 43.

Reply:
done in https://github.com/lyndonbauto/tinkerpop/pull/55/commits/b0c36e9d210b92c91dee4436024fcce0edbeb1ac